src/slice/slice_test.go

```diff
@@ -0,0 +1,20 @@
+//go:build unit
+// +build unit
+
+package slice
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsStringInSlice(t *testing.T) {
+	t.Run("Not Found", func(t *testing.T) {
+		assert.False(t, IsStringInSlice("foo", []string{"bar", "whatever"}))
```

Comment:
whats happens if you pass one nil slice? @esequielvirtuoso

Reply:
It will return false, since the string is not found in slice.
![image](https://user-images.githubusercontent.com/20318955/166066932-777c41d7-a287-42a0-8275-911cb0a25924.png)

Reply:
Note: This test does not exist and the method is private.
I just set them to Private to print the result and answer your question.